### ISSUE TITLE
Cannot find module 'melanke-watchjs'

### ISSUE THREAD

**Issue by thompsonemerson** [TamkeenLMS/electron-window-manager#37]
Hello,
The module is displaying the following error:
```shell
Error: Cannot find module 'melanke-watchjs'
```
The solution is to change the dependency module name on require:
```js
// index.js

// from
const WatchJS = require('melanke-watchjs');

// to
const WatchJS = require('watchjs');
```
Ps.: Probably you upgraded to npm * (version 1.0.4) * but did not upgrade to github * (version 1.0.1) *, therefore it is not possible to send pull request.

**Comment by carlosxjm** [comment]
Look good!

**Comment by carlosxjm** [comment]
I need this!

**Comment by richstandbrook** [comment]
Not sure why we can't do a pull request to fix this? #46

**Comment by thompsonemerson** [comment]
@richstandbrook because the github(1.0.1) version is not the same as npm(1.0.4).

**Comment by richstandbrook** [comment]
Ok, sorry but what are you referring to by github(1.0.1)?

**Comment by thompsonemerson** [comment]
The project version in [github](https://github.com/TamkeenLMS/electron-window-manager/blob/master/package.json#L3)

**Comment by richstandbrook** [comment]
Oh, how has that happened?

**Comment by thompsonemerson** [comment]
It probably updated the npm and did not update the github 🤕

**Comment by richstandbrook** [comment]
Oh! Sorry I assumed npm just pointed to the GitHub repo. I didn't realise it was it's own repo. Well thats annoying!

**Comment by richstandbrook** [comment]
> The development of this module WILL be continued, so feel safe to use it and know I got your back ;)

https://github.com/TamkeenLMS/electron-window-manager/blob/master/README.md#final-notes 🙄

**Comment by a-sync** [comment]
Fixes got merged, and released to npm as v1.0.5 (https://github.com/TamkeenLMS/electron-window-manager/tree/v1.0.5).

The github repo was actually way ahead of npm, but the version in the package json never got changed here.
